bugbot/rules/crash_small_volume.py

```diff
@@ -122,6 +122,9 @@ def handle_bug(self, bug, data):
             ),
             "keywords_to_remove": keywords_to_remove,
             "signatures": signatures,
+            "needinfo_ids": self.get_needinfo_topcrash_ids(bug)
+            if "topcrash" in keywords_to_remove
+            else [],
```

Comment:
We could make it a bit clearer:
```suggestion
            "needinfo_ids": (
                self.get_needinfo_topcrash_ids(bug)
                if "topcrash" in keywords_to_remove
                else []
            ),
```

Reply:
Fixed in 1ae99e364122b2c0b327c4d537cc95f8b7c706e4!